src/core_plugins/tile_map/public/kibana_map.js

```diff
@@ -309,15 +308,8 @@ export class KibanaMap extends EventEmitter {
 
     const southEast = bounds.getSouthEast();
     const northWest = bounds.getNorthWest();
-    let southEastLng = southEast.lng;
-    if (southEastLng > 180) {
-      southEastLng -= 360;
```

Comment:
This trimming functionality does not return the correct bounding box when at a high zoom level. The returned value has a bottom_right that is to the left of top_left. In the original commit, this function was enhanced with a `isTrimmed` parameter so both cases could be satisfied and original consumers did not have to change their usage. Should I add the `isTrimmed` parameter back?